irispy/new_sji.py

```diff
@@ -286,6 +286,12 @@ def dimensions(self):
     def world_axis_physical_types(self):
         return self.cube_like_world_axis_physical_types
 
+    def plot(self, axes=None, plot_axis_indices=None, axes_coordinates=None,
+             axes_units=None, data_unit=None, **kwargs):
```

Comment:
This method needs a docstring so it's clear exactly how it works.  Without one, it will be confusing as it behaves differently to ```NDCubeSequence.plot```.  You can model the docstring of the ```NDCubeSequence.plot_as_cube``` docstring.  But you can simplify it as ```IRISMapCubeSequence``` has a fixed number of dimensions.